### ISSUE TITLE
Enable AXE tool in the CI/CD process

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#17729]
### Description

As part of the company-wide effort to have RH products and services accessible, we plan to enable the AXE tool for critical and severe a11y violations.
